### DISCUSSION TITLE
tailwind typography

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [gregsullivan/_tw#73] [Q&A] [answered]
Hello Greg!

Please clarify me, i wanna my h2 elements to be orange in darkmode, how can i quickly apply it? I noticed that tailwind typografy is not working out of the box or im wrong?

**Comment by gregsullivan** [comment, marked as answer]
Hi Richard—Thanks for posting this!

Are the `h2` elements you're looking to style part of content entered into the block editor, or are they part of a PHP template?

Tailwind Typography is included, but it is only applied within content blocks from the block editor. If those are the styles you'd like to modify, the element modifiers can be found in `./theme/functions.php` (near the top of the file).
